doc/interfaces/viz_flow.rst

```diff
@@ -0,0 +1,163 @@
+.. _viz_flow:
+
+====================================
+Visualization Command Line Interface
+====================================
+
+This section talks about Horizon workflows in DIPY and how to use them.
+
+To follow this tutorial let's make sure we have the latest version of dipy and
+fury installed on our system. ::
+
+  pip install dipy --upgrade
+  pip install fury --upgrade
+
+
+Let's explore the options that horizon provides. ::
+
+  dipy_horizon --help
+
+
+-------------------------
+Visualize 3D Brain Image
+-------------------------
+
+This tutorial provides a basic example of loading an dMRI image to horizon using
+the command line interface.
+
+Using a terminal, let's download a dataset called ``mni_template``. You can 
+skip this step if you already have the dataset downloaded. ::
+
+  dipy_fetch mni_template
+
+To see more details about ``dipy_fetch`` you can refer to :ref:`data_fetch`
+
+This command will download the data in your ``.dipy`` folder placed in your home 
+directory. 
+
+Let's try to load the image.
+
+**For macOS and Linux** ::
+
+  dipy_horizon ~/.dipy/mni_template/mni_icbm152_t1_tal_nlin_asym_09a.nii ~/.dipy/mni_template/mni_icbm152_t2_tal_nlin_asym_09a.nii
+
+**For Windows(cmd)** ::
+  
+  dipy_horizon %USERPROFILE%/.dipy/mni_template/mni_icbm152_t1_tal_nlin_asym_09a.nii %USERPROFILE%/.dipy/mni_template/mni_icbm152_t2_tal_nlin_asym_09a.nii
+
+**For Windows(powershell)** ::
+  
+  dipy_horizon $home/.dipy/mni_template/mni_icbm152_t1_tal_nlin_asym_09a.nii $home/.dipy/mni_template/mni_icbm152_t2_tal_nlin_asym_09a.nii
+
+You can also use your own data by invoking the command below. ::
+
+  dipy_horizon <your_file_name>.nii
+
+
+We also support direct visualization of compressed NIFTI files with extension 
+``.nii.gz``. ::
+
+  dipy_horizon <your_fist_file_name>.nii.gz
+
+
+------------------------------------
+Visualize 3D Brain Image with Volume
+------------------------------------
+
+This tutorial shows how visualize a 3D image with Volume.
+
+Using a terminal, let's download a dataset called ``stanford_hardi``. You can 
+skip this step if you already have the dataset downloaded. ::
+
+  dipy_fetch stanford_hardi
+
+This dataset has a NIFTI file with volumes.
+
+Let's try to load the image.
+
+**For macOS and Linux** ::
+
+  dipy_horizon ~/.dipy/stanford_hardi/HARDI150.nii.gz
+
+**For Windows(cmd)** ::
+  
+  dipy_horizon %USERPROFILE%/.dipy/stanford_hardi/HARDI150.nii.gz
+
+**For Windows(powershell)** ::
+  
+  dipy_horizon $home/.dipy/stanford_hardi/HARDI150.nii.gz
+
+
+--------------------------
+Visualize Brain Tractogram
+--------------------------
+
+This tutorial shows how to visualize a tractogram.
+
+Using a terminal, let's download a dataset called ``bundle_atlas_hcp842``. You
+can skip this step if you already have the dataset downloaded. ::
+
+  dipy_fetch bundle_atlas_hcp842
+
+
+Horizon supports below mentioned tractogram formats.
+
+* .trk
```

Comment:
- there are much more: `'.trk', '.tck', '.trx', '.vtk', '.vtp', '.fib', '.dpy'`
- Why there is a space between `*`and `.trk` ?

Reply:
The problem is viz workflow does not support handling .vtk and .vtp and .fib files